jobs/ml-ops/terraform/container.tf

```diff
@@ -12,7 +12,7 @@ resource "scaleway_container" "inference" {
   cpu_limit      = 2000
   memory_limit   = 2048
   min_scale      = 1
-  max_scale      = 5
+  max_scale      = 1
```

Comment:
We need to preserve the container state as we load the model in memory. Having many instances would lead possibly to ones with no model loaded in memory so they would fail when requested for inference.